cmd/juju/application/deployer/charm.go

```diff
@@ -186,7 +186,6 @@ func (d *deployCharm) deploy(
 			URL:    id.URL,
 			Origin: id.Origin,
 		},
-		d.csMac,
```

Comment:
csMac can also be removed from the deployCharm struct now, deployer/charm.go ln 42